src/suggestion-box.ts

```diff
@@ -0,0 +1,150 @@
+import { LitElement,html,css } from "lit";
+import { customElement,property,state } from "lit/decorators.js";
+
+@customElement("aalam-sgn-box")
+export class AalamSuggestionBoxElement extends LitElement {
+    constructor() {
+        super();
+        this.input.addEventListener("keydown",this.handlearrow.bind(this));
```

Comment:
this.input wouldn't be assigned at this point. Its better to add listeners at the connectedCallback() as thats when the element will get added to the document.